cppcache/src/PdxTypeRegistry.cpp

```diff
@@ -89,8 +89,8 @@ int32_t PdxTypeRegistry::getPDXIdForType(std::shared_ptr<PdxType> nType,
     typeId = cache_->getSerializationRegistry()->GetPDXIdForType(pool, nType);
     nType->setTypeId(typeId);
     pdxTypeToTypeIdMap_.insert(std::make_pair(nType, typeId));
```

Comment:
Perhaps for readability we dispense with the old `insert` with `std::make_pair` and use `emplace`:
```C++
pdxTypeToTypeIdMap_.emplace(nType, typeId);
typeIdToPdxType_.emplace(typeId, nType);
```

Reply:
done!